libafl_bolts/examples/llmp_test/main.rs

```diff
@@ -180,7 +178,7 @@ fn main() -> Result<(), Box<dyn std::error::Error>> {
         }
         "ctr" => {
             let mut client =
-                llmp::LlmpClient::create_attach_to_tcp(StdShMemProvider::new()?, port)?;
+                llmp::LlmpClient::create_attach_to_tcp(StdShMemProvider::new()?, port, 1337)?;
```

Comment:
Why hard code this value?

Reply:
It's an example code, not test code, people are free to use this example to build their own LLMP stuff

Reply:
Plus here won't it be an issue if more clients join with the same ID? I guess they don't get removed anymore?